src/dbm/DBMPrediction.vb

```diff
@@ -67,43 +56,46 @@ Namespace Vitens.DynamicBandwidthMonitor
     End Sub
 
 
-    Public Sub Calculate(Values() As Double)
+    Public Shared Function Calculate(Values() As Double) As DBMPrediction
 
-      ' Calculates and stores prediction and control limits by removing
+      ' Calculates and prediction and control limits by removing
       ' outliers from the Values array and extrapolating the regression
       ' line by one interval.
+      ' The result of the calculation is returned as a new object.
 
       Dim Statistics As New DBMStatistics
-      Dim ControlLimit As Double

```

Comment:
@mkbosmans What is the reason for removing this declaration and moving it to https://github.com/Vitens/DBM/pull/40/files#diff-7b8c2ff8fc0741f10baa46cf4d9fa404R86?

Reply:
Nothing really important, it's just that I'm used to declaring variables close to where they are used, as is normally done in e.g. C++/C#.
In this case declaring the ControlLimit variable upon first assignment makes sure that using it outside the With block is an error that is catched by the compiler.